### PR TITLE
ENTESB-14422 - remove DV and related teiid components from fuse install for future 7.x releases, starting in 7.8

### PR DESCRIPTION
**Pull request by vhalbert**

### CODE REVIEW

**Comment by gashcrumb** [comment]
LGTM but I wouldn't mind a second pair of eyes, @phantomjinx would you mind having a review also?

File: templates/fuse-online-template.yml [hunk 1]
```diff
@@ -1988,300 +1987,3 @@ objects:
...
-    name: fuse-dv
-  spec:
-    tags:
-    - from:
-        kind: DockerImage
-        name: ${SYNDESIS_REGISTRY}/fuse7-tech-preview/fuse-dv:1.5
-      importPolicy:
-        scheduled: true
-      name: "1.5"
-parameters:
```
Comment (phantomjinx) [inline templates/fuse-online-template.yml:2127]: Think you've taken too much away here. These parameters should remain.

Comment (vhalbert) [reply templates/fuse-online-template.yml:2127]: As for what was missed, that's on the master branch, just dealing with the 7.8 release for now.   Master here and syndesis is still to come.

Comment (vhalbert) [reply templates/fuse-online-template.yml:2127]: As for the import policy, that's related to the fuse-dv image,  that on its own, wouldn't apply to anything.

Comment (vhalbert) [reply templates/fuse-online-template.yml:2127]: @phantomjinx  ok, added back the parameters and made the other changes u inidicated.

**Pull request merged**